src/vm/jitinterface.cpp

```diff
@@ -9953,8 +9953,8 @@ void* CEEInfo::getPInvokeUnmanagedTarget(CORINFO_METHOD_HANDLE method,
 
 /*********************************************************************/
     // return address of fixup area for late-bound N/Direct calls.
-void* CEEInfo::getAddressOfPInvokeFixup(CORINFO_METHOD_HANDLE method,
-                                        void **ppIndirection)
```

Comment:
getFunctionEntryPoint returns the managed entrypoint. It is basically the result of ldftn `<method>`. It is valid for any method. For P/Invoke methods, it is address of the PInvoke marshalling stub.

 getAddressOfPInvokeFixup returns the unmanaged PInvoke target. It is the address of the actual unmanaged method that the PInvoke calls (e.g. `kernel32!ReadFile`).